logical_router.go

```diff
@@ -145,3 +146,104 @@ func (odbi *ovnDBImp) GetLogicalRouters() ([]*LogicalRouter, error) {
 
 	return listLR, nil
 }
+
+func (odbi *ovnDBImp) lrlbAddImp(lr string, lb string) (*OvnCommand, error) {
+	var operations []libovsdb.Operation
+	row := make(OVNRow)
+	row["name"] = lb
+	lbuuid := odbi.getRowUUID(tableLoadBalancer, row)
+	if len(lbuuid) == 0 {
+		return nil, ErrorNotFound
+	}
```

Comment:
Can you add check for lr too?

Reply:
@vtolstov : Also added for LSwitch too :)

Reply:
ok